### PR TITLE
Add notes for Deprecating Rackspace CR

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by shiramax** [comment]
@tbrisker can you please review?

File: _includes/manuals/2.0/1.2_release_notes.md [hunk 1]
```diff
@@ -421,6 +421,7 @@ External authentication sources can now be assigned to taxonomies using a new UI
 *A full list of changes in 2.0.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1132)*
 
 ### Deprecations
+* Deprecate Rackspace Compute Resource ([#29285](https://projects.theforeman.org/issues/29285))
```
Comment (tbrisker) [inline _includes/manuals/2.0/1.2_release_notes.md]: Please add a bit more information beyond just the RM title, and maybe link to the discourse post?

Comment (shiramax) [reply _includes/manuals/2.0/1.2_release_notes.md]: thanks @tbrisker I've added the link, but I'm not sure which more information is needed here.

File: _includes/manuals/2.0/1.2_release_notes.md [hunk 2]
```diff
@@ -421,6 +421,7 @@ External authentication sources can now be assigned to taxonomies using a new UI
 *A full list of changes in 2.0.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1132)*
 
 ### Deprecations
+* Deprecate Rackspace Compute Resource from foreman core ([#29285](https://projects.theforeman.org/issues/29285))  [Further details](https://community.theforeman.org/t/dropping-rackspace-compute-resource-from-foreman-core/17690)
```
Comment (tbrisker) [inline _includes/manuals/2.0/1.2_release_notes.md]:
```suggestion
* Support for Rackspace as a Compute Resource is deprecated and will be removed in Foreman 2.1.  [Further details](https://community.theforeman.org/t/dropping-rackspace-compute-resource-from-foreman-core/17690)
```

Comment (shiramax) [reply _includes/manuals/2.0/1.2_release_notes.md]: done :)

**Review by tbrisker — APPROVED** [review]
Thanks @shiramax !

**Pull request merged**